roles/rciam-metrics/defaults/main.yml

```diff
@@ -0,0 +1,31 @@
+# file: group_vars/metrics/main.yml
+#
+---
+
+metrics_path: /srv/rciam-metrics-client/rciam-metrics
+metrics_repo: https://github.com/lionick/rciam-metrics-dev
+metrics_latest_release: rc-example
+metrics_version: main
+metrics_fast_api_folder_name: app
+metrics_api_url: http://localhost:8004/
+metrics_db: metrics-dev
+metrics_db_admin: rciam
+metrics_db_password: secret
+metrics_db_hostname: db.example.com
+metrics_user:
+  name: rciam-metrics-client
+  path: /srv/rciam-metrics-client
+  group: rciam-metrics-client
+  gecos: "RCIAM Metrics Client Name"
+  shell: /bin/bash
+#metrics_symlinks:
+#  - target: "/path/to/target"
+#    link: "/path/to/symlink"
+
+# OIDC Client
+oidc_client_id: client-id
+oidc_client_secret: client-secret
+oidc_client_issuer: client-issuer
+oidc_client_connect_url: client-connect-url
+oidc_client_idtoken_model: client-idtoken-model
+oidc_client_redirect_uri: client-redirect-uri
```

Comment:
Each tenant needs to have its own OIDC configuration so I'd suggest changing this to:
```
metrics_oidc:
  tenant1:
    client_id: client-id
    client_secret: client-secret
    client_redirect_uri: https://metrics.rciam.example.org/callback
    issuer: https://tenant1.rciam.example.org/auth/realms/rciam
  #tenant2:
  #  client_id: client-id
  #  client_secret: client-secret
  #  client_redirect_uri: https://metrics.rciam.example.org/callback
  #  issuer: https://tenant2.rciam.example.org/auth/realms/rciam
```